src/utils/customer-data.js

```diff
@@ -1,17 +1,25 @@
-const ALLOWED_DATA_TYPES = ['string', 'number', 'boolean'];
 const CUSTOMER_DATA_CHARS_LIMIT = 1000;
 
-export const parseProvidedData = (data) => {
-  if (typeof data === 'object' && data !== null && !Array.isArray(data)) {
-    const parsedData = Object.keys(data).reduce((collection, dataKey) => {
-      if (ALLOWED_DATA_TYPES.includes(typeof data[dataKey]) || data[dataKey] === null) {
-        collection[dataKey] = data[dataKey];
-      }
+const filterProvidedData = (data) => {
+  return Array(5).reduce((obj, cv, currentIndex) => {
```

Comment:
This is preferred over the constructor since it is easier to read, it makes it easy to understand that its an array of size 5 and not an array with just an element of value 5 
```suggestion
  return Array.from({length: 5}).reduce((obj, cv, currentIndex) => {
```

Reply:
👍